lib/responses.js

```diff
@@ -524,12 +524,28 @@ function deleteRouteFromDB(req, res) {
   }, res);
 }
 
+/**
+ * Forces a redirect back to the non-published
+ * version of a uri
+ *
+ * @param {Number} code
+ * @returns {Function}
+ */
+function forceDefault(code = 303) {
```

Comment:
I feel this function name is confusing. "Canonical" seems more correct to me than "default"

Reply:
Hmm I disagree as it's simply a redirect back the the editable/default page. I could go for `editable`, but this is not what I would call a canonical route. 

I'll change to `editable`